openstack/sharedfilesystems/v2/shares/results.go

```diff
@@ -2,12 +2,13 @@ package shares
 
 import (
 	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
 )
 
 // Share contains all information associated with an OpenStack Share
 type Share struct {
 	// The availability zone of the share
-	AvailabilityZone string `json:"availability_zone"`
+	AvailabilityZone string `json:"availability_zone,omitempty"`
```

Comment:
Not sure if this is a good idea to have each of the fields as `omitempty`. It allows us to re-use the type for short and long listing though. Open to suggestions on better alternatives